metacat-main/src/main/java/com/netflix/metacat/main/services/impl/PartitionServiceImpl.java

```diff
@@ -156,27 +165,29 @@ public PartitionServiceImpl(
                 uris.add(partitionDto.getDataUri());
             });
 
-            registry.gauge(this.partitionGetCountId.withTags(new HashMap<>(name.parts())), result.size());
+            registry.gauge(this.partitionGetGaugeId.withTags(new HashMap<>(name.parts())), result.size());
```

Comment:
Here I would use `Maps.newHashMap(name.parts())` for convenience of not having "new" in the code